controllers/lmes/lmevaljob_controller.go

```diff
@@ -198,6 +202,11 @@ func (r *LMEvalJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		return r.handleComplete(ctx, log, job)
 	case lmesv1alpha1.CancelledJobState:
 		return r.handleCancel(ctx, log, job)
+	case lmesv1alpha1.SuspendedJobState:
+		if !job.Spec.Suspend {
+			return r.handleResume(ctx, log, job)
+		}
+		return ctrl.Result{}, nil
```

Comment:
you can remove this and fall through to the end of the function, there is a final no-op return there.